### ISSUE TITLE
[DOCS] : Update readme

### ISSUE THREAD

**Issue by PritamSarbajna** [PritamSarbajna/tourism-website#3]
update the readme with latest previews.

**Comment by Seyi-Ojo** [comment]
Hi, I saw your project and wondered if I could work on it. But first, could you please elaborate on what I should do?

**Comment by PritamSarbajna** [comment]
Hey @Seyi-Ojo this is a frontend project. Do you need any help to get started with this project ?? or are you searching for any issue to work on ?

**Comment by Seyi-Ojo** [comment]
Hello @PritamSarbajna Yes please, I need an issue to work on.

**Comment by PritamSarbajna** [comment]
Hey @Seyi-Ojo you can choose this issue it needs to only update the readme with the current screenshot of the website. also do you know how to make a PR and fork and all that If you don't or any other help feel free to ask me. Also here is my discord username **Pritam_Sarbajna#8032**

**Comment by Seyi-Ojo** [comment]
Thank you very much Pritam, I know how to make a PR and fork but I would still ask for help when I need to, thank you so much.

**Comment by PritamSarbajna** [comment]
@Seyi-Ojo yeah sure and if you're beginner in development and open source then I think this repo can be best for you !

**Comment by Techkrish1** [comment]
Can you assign me to improve the README.md?

**Comment by Seyi-Ojo** [comment]
Hey Pritam, so I checked the README.md file and saw that I am to add a screenshot, how do I go about this? I am trying to learn with this, so I need you to explain what I should do. Thank you.

**Comment by Seyi-Ojo** [comment]
Hey Pritam, so I checked the README.md file and saw that I am to add a screenshot, how do I go about this? I am trying to learn with this, so I need you to explain what I should do. Thank you.

**Comment by PritamSarbajna** [comment]
Don't use similar comment

**Comment by PritamSarbajna** [comment]
Hey @Seyi-Ojo the readme needs update because the website's UI has been changed

**Comment by Seyi-Ojo** [comment]
Okay, I will redo it @PritamSarbajna

**Comment by PritamSarbajna** [comment]
Yeah sure @Seyi-Ojo

**Comment by pce-amishapatil** [comment]
Hi, I saw your project if I would like to work on it. could you please elaborate on what I should do?

**Comment by Shreyashkshirsagar** [comment]
I believe that improving the documentation in the README file will not only enhance the project's usability but also make it more accessible to newcomers and existing users. My experience in documentation and attention to detail would be beneficial in ensuring that the README is clear and informative.
Before I proceed, I wanted to confirm whether you would be comfortable with me taking on this task. If so, I will start working on it promptly and ensure that the updated documentation aligns with the project's objectives.

**Comment by PritamSarbajna** [comment]
Yeah @Shreyashkshirsagar, work on it 🚀
